modules/rostests/apitests/powrprof/power.c

```diff
@@ -4620,7 +4620,8 @@ pPP_original->user.OptimizeForPowerDc=1;
    ret = ValidatePowerPolicies(&gpp,&pp);
    ok(ret, "function was expected to succeed error %i\n",(UINT)GetLastError());
    ok(globalcompare(gpp,gpp_compare),"Difference Found\n");
-   pp_compare.user.MaxSleepAc=4;
+   // FIXME: Duplicate line.
+   // pp_compare.user.MaxSleepAc=4;
```

Comment:
Considering the code above where variable is initialized, you can safely delete this line.

Reply:
Deleted.